src/git-extract

```diff
@@ -51,20 +55,18 @@ function preconditions {
 
 
 function steps {
-  if [ "$initial_open_changes" = true ]; then
-    echo "stash_open_changes"
-  fi
+  local has_remote="$(has_remote_url)"
```

Comment:
how about setting a global `$has_remote` in `src/helpers/helpers.sh`? Especially with the coming changes it seems like that would be useful,